logging/interceptor.go

```diff
@@ -80,23 +80,49 @@ func UnaryClientInterceptor(logger *logrus.Logger, opts ...Option) grpc.UnaryCli
 		fields[DefaultGRPCCodeKey] = code.String()
 
 		levelLogf(
-			logrus.NewEntry(logger).WithFields(fields),
+			entry.WithFields(fields),
 			options.codeToLevel(code),
-			"finished unary call with code "+code.String())
+			fmt.Sprintf("finished unary call with code %s", code.String()))
```

Comment:
Per definition of levelLogf ```func levelLogf(entry *logrus.Entry, level logrus.Level, format string, args ...interface{}) {``` we can add arguments directly into this function instead of using fmt.Sprintf